Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts
```

Comment:
node changes quite frequently. should we pin the version here to not run into unexpected issues?

Reply:
Yeah we could do that, latest stable version ?